### DISCUSSION TITLE
Seja bem vindo-vinda(o) às discussões

### DISCUSSION THREAD

**Discussion by jjesusfilho** [jjesusfilho/tjsp#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by brittog** [comment]
Companheiro, uma dúvida (sou bem leigo, então perdoe se a resposta for óbvia). Os resultados de busca no TJSP vêm com um pedaço da ementa truncado, precisando clicar no "+" para visualizar a ementa completa. Teria como, na hora de baixar os resultados, esse procedimento ser realizado automaticamente para baixarmos a ementa completa? Muito obrigado e parabéns pelo trabalho!

**Reply by giuliacarv** [reply]
Tenho essa dúvida também
